extensions/fbc/src/org/sbml/jsbml/ext/fbc/converters/FbcV1ToCobraConverter.java

```diff
@@ -53,13 +55,129 @@
+    if (lowerBoundParameter == null) 
+    {
+      lowerBoundParameter = kineticLaw.createLocalParameter(CobraConstants.LOWER_BOUND);
+      if (this.defaultLowerBound != null)
+      {
+        lowerBoundParameter.setValue(this.defaultLowerBound);  
+      } else 
+      {
+        lowerBoundParameter.setValue(Double.NEGATIVE_INFINITY);
+      }
+      lowerBoundParameter.setExplicitlyConstant(true);
+      lowerBoundParameter.setUnits(CobraConstants.mmol_per_gDW_per_hr);
+    }
+    
+    LocalParameter objectiveCoefficientParameter = kineticLaw.getLocalParameter(CobraConstants.OBJECTIVE_COEFFICIENT);
+    if (objectiveCoefficientParameter == null)
+    {
+      objectiveCoefficientParameter = kineticLaw.createLocalParameter(CobraConstants.OBJECTIVE_COEFFICIENT);
+      objectiveCoefficientParameter.setUnits(CobraConstants.mmol_per_gDW_per_hr);
+      objectiveCoefficientParameter.setValue(0d);
+    }
+  }
+    
+  private void unsetConstantForSpeciesReferencesOfReaction(Reaction reaction) {
+    // unset attribute constant for products and reactants of reaction
+    for (SpeciesReference speciesReference: reaction.getListOfProducts()) {
+      speciesReference.unsetConstant();
+    }
+    for (SpeciesReference speciesReference: reaction.getListOfReactants()) {
+      speciesReference.unsetConstant();
+    }
+  }
+   
+  private void updateKineticLawFromBound (Reaction reaction, FluxBound bound)
+  {
+    if (reaction == null || bound == null) {
+      return;
+    }
+    Operation operation = bound.getOperation();
+    KineticLaw kineticLaw = reaction.getKineticLaw();
+    LocalParameter lowerBoundParameter = kineticLaw.getLocalParameter(CobraConstants.LOWER_BOUND);
+    LocalParameter upperBoundParameter = kineticLaw.getLocalParameter(CobraConstants.UPPER_BOUND);
+    
+    if (operation.equals(FluxBound.Operation.LESS_EQUAL) || operation.equals(FluxBound.Operation.EQUAL)) 
+    {
+      upperBoundParameter.setValue(bound.getValue());
+    }
+    
+    if (operation.equals(FluxBound.Operation.GREATER_EQUAL) || operation.equals(FluxBound.Operation.EQUAL)) 
+    {
+      lowerBoundParameter.setValue(bound.getValue());
+    }
+  }
+   
+  private void setObjectiveCoefficient(FBCModelPlugin plugin, Model model) {
+    if (plugin == null || model == null) {
+      return;
+    }
+    Objective obj = plugin.getActiveObjectiveInstance();
+    if (obj == null) {
+      return;
+    }
+    for (FluxObjective fluxObj : obj.getListOfFluxObjectives()) {
+      if (fluxObj == null) {
+        continue;
+      }
+      Reaction reaction = model.getReaction(fluxObj.getReaction());
+      if (reaction == null) {
+        continue;
+      }
+      KineticLaw law = reaction.getKineticLaw();
+      if (law == null) {
+        continue;
+      }
+      LocalParameter param = law.getLocalParameter(CobraConstants.OBJECTIVE_COEFFICIENT);
+      param.setValue(fluxObj.getCoefficient());
+    }
+  }
```

Comment:
Update the objective coefficient value for the reactions used in the active objective. This is being added for https://www.pivotaltracker.com/story/show/161591970